lib/rspec/rails/view_rendering.rb

```diff
@@ -39,18 +39,50 @@ def render_views?
         self.class.render_views? || !controller.class.respond_to?(:view_paths)
       end
 
-      # Delegates find_all to the submitted path set and then returns templates
-      # with modified source
-      #
-      # @private
-      class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+      class EmptyTemplateResolverFactory
```

Comment:
When we build thinks like this we normally make them "instance less" to avoid unnecessary allocations, please refactor to a static instance, or better yet a module method. Can I also suggest a better name is desired? Something like `DecoratePath` or `BuildEmptyTemplateResolver`